.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,7 @@ Eclipse RDF4J follows the [Eclipse Coding Conventions for Java](https://wiki.ecl
 
 - We use a line width of 120 characters.
 - We use Unix line endings (LF).
+- We require curly braces for every control statement body (e.g. if-else), even if it is a single line.
```

Comment:
Not part of this issue, strictly speaking, but I just realized it might be good to mention this here.